### ISSUE TITLE
7900 xtx

### ISSUE THREAD

**Issue by texport** [Vanilla-OS/iso-builder#125]
### Issue Description

I'm trying to install vanilla with a 7900 xtx graphics card, after I press try vanilla, I get a black screen. In safe mode, the same.

### Steps to Reproduce

Try to run vanilla with a usb flash drive with a video card 7900 xtx

### On what version of Vanilla OS this happens?

22.10

**Comment by ghost** [comment]
The gpu might be too new for the kernel to have drivers for it

**Comment by kbdharun** [comment]
> The gpu might be too new for the kernel to have drivers for it

Yeah, this is highly likely as we currently use 5.19 kernel I think

**Comment by texport** [comment]
in general, I tried pop os and fedora, pop os turned out to run on a tripled graphics, to start 7900 xtx I did according to this article https://catwithcode.github.io/Blog/2022.12.16_Radeon_7900XT-XTX_On_Linux/Radeon_7900XT-XTX_On_Linux.html, but for some reason the cooler on the video card did not work. In Fedora, it turned out only through safe mode, but the same thing works crookedly, the hertz on the monitor is not set above 60 (

when will vanilla os support new graphic cards?)

**Comment by kbdharun** [comment]
> when will vanilla os support new graphic cards?)

We are working on custom kernel signing in #dev, so soon you will start seeing newer kernel versions in the base.

**Comment by texport** [comment]
when will that happen?)

**Comment by kbdharun** [comment]
> when will that happen?)

In future maybe after 23.04, but definitely not now I think.
